installer/Migration/V5_1_0/lang-string/recruitment.yaml

```diff
@@ -95,4 +95,39 @@ langStrings:
   -
     value: 'Should be the current date or a previous date'
     unitId: should_be_current_date_previous_date
+  -
+    value: 'Application Stage'
+    unitId: application_stage
+  -
+    value: 'Reject'
+    unitId: reject
```

Comment:
`Reject` is already defined in `5.0` general.
@SharukaNP, need a unit test to identify if language strings are duplicated in different groups.